backend/internal/repository/hostGroup.go

```
@@ -0,0 +1,129 @@
+	DeleteHostGroup(ctx context.Context, id int) error
+}
+
+// SQL Queries (embed the .sql files)
+
+var getHostGroupByIDQuery string
+
+var getAllHostGroupsQuery string
+
+var createHostGroupQuery string
+
+var updateHostGroupQuery string
```

golangci-lint (1.64.8)

var `updateHostGroupQuery` is unused

(unused)